src/repl-frontend/scala/tools/nsc/interpreter/shell/JavapClass.scala

```diff
@@ -331,7 +331,7 @@ object Javap {
   private def massage(arg: String): Seq[String] = {
     require(arg startsWith "-")
     // arg matches opt "-foo/-f" if prefix of -foo or exactly -f
-    val r = """(-[^/]*)(?:/(-.))?""".r
+    val r = """(-[[^-/]|[a-z]]*)(?:/(-[a-z]))?""".r
```

Comment:
this is just `helps.map(_.split("/"))`, except it is easier to cope with second element missing. If this is a good idea, might as well require `-[a-z]` on both sides. The alternation is especially useless because `[a-z]` is subsumed by `[^-/]` which means not a hyphen or slash.

Reply:
Making the change now.